### DISCUSSION TITLE
Clear or Empty the entire List

### DISCUSSION THREAD

**Discussion by nnyerges** [nkaaf/Arduino-List#13] [Ideas]
Hello, very nice and useful library for Arduino.
It will be great to add a method to the AbstractList class to empty or clear a List, something like this: "**.clear(void)**".
Currently the only way I see to empty the entire list is to use "**.remove(index)**" inside a loop for all the items in the list.
If there is currently another way to do it, please let me know.
Have a nice day.

**Comment by nkaaf** [comment]
Hello, thank you very much for your feedback. This is a really good idea. At the moment there is the possibility only via a for loop in which `remove(index)` is called. I take the idea and I will implement a `clear()` method.
Have a nice day, too.

**Comment by nkaaf** [comment]
I have created a release ([2.1.0](https://github.com/nkaaf/Arduino-List/releases/tag/v2.1.0)) that includes this feature. In 24 hours at the latest the new version should be published in the Arduino IDE. Thanks again for your contribution.

**Reply by nnyerges** [reply]
Excelent!

It is always a pleasure to contribute to positive things and especially if they make life easier for others.

Anyway, thanks to you! for adding the method so fast... whooo! Great 👍🏻.
